test/helpers.js

```diff
@@ -66,6 +66,8 @@ function testSingle(zlib_method, pako_method, data, options) {
   // position (= no additional gzip headers used)
   if (options.ignore_os) zlib_result[9] = pako_result[9];
 
+  zlib_result = Array.from(Uint8Array);
+  pako_result = Array.from(Uint8Array);
```

Comment:
1. I don't understand why this should work at all. Looks like data will be missed.
2. I would cast to typed array. Ordinary arrays are legacy for ancient browsers, id doesn't worth spread those without need.

Reply:
Yes, it's kind of urgent. We need pako to pass with Node.js 12. It would be great to have it fixed ASAP.